src/codegeneration/PlaceholderParser.js

```diff
@@ -20,7 +20,6 @@ import {
 } from '../syntax/trees/ParseTreeType.js';
 import {IdentifierToken} from '../syntax/IdentifierToken.js';
 import {LiteralToken} from '../syntax/LiteralToken.js';
-import {Map} from '../runtime/polyfills/Map.js';
```

Comment:
Does this create a possibly implicit dependence on browser, browser version or V8 version?